nautobot/docs/user-guides/getting-started-nautobot-ui.md

```diff
@@ -0,0 +1,606 @@
+# Getting Nautobot Up and Running in the Web UI
+
+## Introduction and Scope
+
+The audience for this user guide is a user new to Nautobot. 
+It is intended to assist the user with:
+* Understanding the prerequisites to adding a new Device
+* Creating the necessary prerequisite objects to add a new Device
+* Adding new Devices
+* Adding and understanding Regions
+* Adding and Understanding Platforms
+* Adding and Understanding Tenants and Tenant Groups
+* Adding Interfaces to a Device
+* Adding VLANs and Understanding VLAN Groups
+* Understanding IP Address Management (IPAM) in Nautobot
+* How to navigate to the object you are interested in
+* How to use the Search Bar to find the object you are interested in
+
+## Requirements
+
+1. A functional Nautobot installation
+
+## Creating Devices in Nautobot
+
+A network Device in Nautobot has a few required attributes:
+* A Device Role
+* A Device Type
+  * A Device Type requires a Manufacturer
+* A Site
+
+Looking at the list above, there are four objects in Nautobot that must be present prior to creating a related Device.
+The following sections will guide you through how to create each object type.
+
+### Creating a Device Role
+
+To create a new Device, you will need an existing Device Role or need to create a new Device Role instance.
+
+To create a new Device Role:
+1. Click on **Devices** in the top navigation menu
+2. Find **Device Roles** on the drop-down menu
+3. Select `+`
+4. In the `Add a new device role` form, populate a Name
+    * The Slug will auto-populate based on the Name, but can be manually overwritten
+5. Click on Create    
+
+![](images/getting-started-nautobot-ui/3-create%20role.png)
+
+### Creating a Manufacturer
+
+To create a new Device Type, you will need an existing Manufacturer or need to create a new Manufacturer instance.
+
+To create a new Manufacturer:
+1. Click on **Devices** in the top navigation menu
+2. Find **Manufacturers** on the drop-down
+3. Select `+`
+4. In the `Add a new manufacturer` form, populate the Name
+     * The Slug will auto-populate based on the Name, but can be manually overwritten
+5. Click on **Create**
+
+![](images/getting-started-nautobot-ui/2-create%20manufacturer.png)
+
+### Create a Device Type
+
+To create a new Device, you will need an existing Device Type or need to create a new Device Type instance.
+
+To create a new Device Type:
+1. Click on **Devices** in the top navigation menu
```

Comment:
Please add a line break before all lists. It is preventing this list from rendering correctly.